x-pack/plugins/ingest_manager/server/services/setup.ts

```diff
@@ -5,14 +5,26 @@
  */
 
 import { SavedObjectsClientContract } from 'kibana/server';
+import { CallESAsCurrentUser } from '../types';
 import { agentConfigService } from './agent_config';
 import { outputService } from './output';
+import { installLatestPackage } from './epm/packages/install';
 
-export async function setup(soClient: SavedObjectsClientContract) {
-  // TODO install default packages
-
+export async function setup(
+  soClient: SavedObjectsClientContract,
+  callCluster: CallESAsCurrentUser
+) {
   await Promise.all([
     outputService.ensureDefaultOutput(soClient),
     agentConfigService.ensureDefaultAgentConfig(soClient),
+
+    // packages installed by default
+    installLatestPackage({
+      savedObjectsClient: soClient,
+      pkgName: 'base',
+      callCluster,
+      internal: true,
+    }),
+    installLatestPackage({ savedObjectsClient: soClient, pkgName: 'system', callCluster }),
```

Comment:
@ruflin if a user uninstalls the ` system` package, should we be installing it every time? Or should they not be allowed to uninstall `system` via the UI?